teps/0091-trusted-resources.md

```diff
@@ -336,39 +262,28 @@ metadata:
   name: config-trusted-resources
   namespace: tekton-pipelines
 data:
-  onError: "stopAndFail"
+  skip-verification: false
   cosign-pubkey-path: "/etc/signing-secrets/cosign.pub"
   kms-pubkey-path: "gcpkms://projects/<project>/locations/<location>/keyRings/<keyring>/cryptoKeys/<key>"
 ```
-`onError: [ stopAndFail | continue ]`. (Optional, default `continue`)
-`stopAndFail` will cause the build to fail if image verification fails,  `continue` will allow the build to continue even
-if verification fails.
 
-`skip-verification`. (Optional, default `true`), if true then directly return nil, if false then do the verfication
+`skip-verification`. (Optional, default `true`), if true then directly return nil, if false then do the verification
 
 `cosign-pubkey-path`. (Optional, default `/etc/signing-secrets/cosign.pub`), it specifies the secret path to store the cosign pubkey
 
 `kms-pubkey-path`.  (Optional, default empty), it specifies the KMS reference
 
-The key path configuration can be used as default keys when no keys are founded in `keyref`.
+The key path configuration can be used as default keys when no keys are found in `keyref`.
 
 
 ### Integrate with Remote Resource Resolution
 
 TEP-0060 introduces a new ResourceRequest reconciler that will actually pull
 the remote image and expand it to a Task.
 
-There are several ways to integrate the Trusted Tesource with Remote Resource Resolution
+This TEP proposed to create the remote resolution request before verification in webhook. All alternatives will be discussed at [alternatives](#alternatives).
```

Comment:
Grammar nit: "...proposes creating the remote resolution request before verification in the webhook."

Second nit: "Alternatives are discussed at..." (I doubt it's ever **all** alternatives! 🙃)

Reply:
yes! Thank you! 😄